test/Microsoft.AspNetCore.SignalR.Tests/WebSocketsTransportTests.cs

```diff
@@ -35,6 +40,33 @@ public WebSocketsTransportTests(ServerFixture<Startup> serverFixture, ITestOutpu
             _serverFixture = serverFixture;
         }
 
+        [ConditionalFact]
+        [OSSkipCondition(OperatingSystems.Windows, WindowsVersions.Win7, WindowsVersions.Win2008R2, SkipReason = "No WebSockets Client for this platform")]
```

Comment:
I'm going to file a bug for this (not worth doing in this PR) but we're going to need a new skip attribute for WebSockets because we actually **do** want to run on Windows 7 but only with `netcoreapp2.1`. They switched `ClientWebSocket` to use the Managed implementation on all OSes in .NET Core 2.1 which means it now supports Windows 7.

Reply:
Filed https://github.com/aspnet/SignalR/issues/1568 to track.